src/rubocopAutocorrectProvider.ts

```diff
@@ -15,8 +15,7 @@ export default class RubocopAutocorrectProvider
     const config = getConfig();
     try {
       const args = [...getCommandArguments(document.fileName), ...additionalArguments];
-      if (additionalArguments.length === 0) args.push('--autocorrect');
```

Comment:
I think this was also a typo, it's meant to be `--auto-correct` instead of `--autocorrect`. See: https://docs.rubocop.org/rubocop/usage/auto_correct.html